### PR TITLE
routing for application

### PR DESCRIPTION
**Pull request by songtv-0006**
Routing and Authenticate for application

### CODE REVIEW

**Comment by songtv-0006** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Login') }}</div>
+
+                <div class="card-body">
+                    <form method="POST" action="{{ route('login') }}">
```
Comment (tranducquoc) [inline resources/views/auth/login.blade.php]: dùng form laravelcolective. fix hết tất cả những file còn lại nhé

**Comment by songtv-0006** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**